cmd/envtool/envtool.go

```diff
@@ -98,16 +99,20 @@ func setupAnyPostgres(ctx context.Context, logger *zap.SugaredLogger, uri string
 		return err
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3452
-
-	var pgPool *pgdb.Pool
+	var pgPool *pgxpool.Pool
 
 	var retry int64
 	for ctx.Err() == nil {
-		if pgPool, err = pgdb.NewPool(ctx, uri, logger.Desugar(), sp); err == nil {
+		var p *pool.Pool
+		if p, err = pool.New(uri, logger.Desugar(), sp); err == nil {
```

Comment:
If `err == nil`, it breaks in L108. I don't think that's what we want, we want L111 to execute.
Could you add simple test please? 🤗

Reply:
nice catch